docs/www/config-reference.md

```diff
@@ -0,0 +1,131 @@
+| raft_heartbeat_interval_ms | Milliseconds for raft leader heartbeats | Optional | 150ms |
+| raft_heartbeat_timeout_ms | raft heartbeat RPC timeout | Optional | 3s |
+| raft_io_timeout_ms | Raft I/O timeout | Optional | 10000ms |
+| raft_replicate_batch_window_size | Max size of requests cached for replication | Optional | 1MB |
+| raft_timeout_now_timeout_ms | Timeout for a timeout now request | Optional | 1s |
+| raft_transfer_leader_recovery_timeout_ms | Timeout waiting for follower recovery when transferring leadership | Optional | 10s |
+| readers_cache_eviction_timeout_ms | Duration after which inactive readers will be evicted from cache | Optional | 30s |
+| reclaim_growth_window | Length of time in which reclaim sizes grow | Optional | 3000ms |
+| reclaim_max_size | Maximum batch cache reclaim size | Optional | 4MB |
+| reclaim_min_size | Minimum batch cache reclaim size | Optional | 128KB |
+| reclaim_stable_window | Length of time above which growth is reset | Optional | 10000ms |
+| recovery_append_timeout_ms | Timeout for append entries requests issued while updating stale follower | Optional | 5s |
+| release_cache_on_segment_roll | Free cache when segments roll | Optional | false |
+| replicate_append_timeout_ms | Timeout for append entries requests issued while replicating entries | Optional | 3s |
+| retention_bytes | max bytes per partition on disk before triggering a compaction | Optional | None |
+| rm_sync_timeout_ms | Time to wait state catch up before rejecting a request | Optional | 2000ms |
+| rm_violation_recovery_policy | Describes how to recover from an invariant violation happened on the partition level | Optional | model::violation_recovery_policy::crash |
+| rpc_server | IpAddress and port for RPC server | Optional | 127.0.0.1:33145) |
+| rpc_server_tls | TLS configuration for RPC server | Optional | tls_config::validate |
+| seed_server_meta_topic_partitions | Number of partitions in internal raft metadata topic | Optional | 7 |
+| seed_servers | List of the seed servers used to join current cluster; If the seed_server list is empty the node will be a cluster root and it will form a new cluster | Optional | None |
+| segment_appender_flush_timeout_ms | Maximum delay until buffered data is written | Optional | 1sms |
+| stm_snapshot_recovery_policy | Describes how to recover from an invariant violation happened during reading a stm snapshot | Optional | model::violation_recovery_policy::crash |
+| superusers | List of superuser usernames | Optional | None |
+| target_quota_byte_rate | Target quota byte rate (bytes per second) (2GB default) | Optional | 2GB |
+| tm_sync_timeout_ms | Time to wait state catch up before rejecting a request | Optional | 2000ms |
+| tm_violation_recovery_policy | Describes how to recover from an invariant violation happened on the transaction coordinator level | Optional | model::violation_recovery_policy::crash |
```

Comment:
What's the default value here?

Reply:
crash if there is an invariant violation, no?